osbuild/util/chroot.py

```diff
@@ -46,4 +46,4 @@ def __exit__(self, exc_type, exc_value, tracebk):
     def run(self, cmd, **kwargs):
         cmd = ["/usr/sbin/chroot", self.root] + cmd
         # pylint: disable=subprocess-run-check
-        subprocess.run(cmd, **kwargs)  # noqa: PLW1510
+        return subprocess.run(cmd, **kwargs)  # noqa: PLW1510
```

Comment:
This is the change.

Reply:
please split it into a separate commit.

Reply:
This returns a `subprocess.CompletedProcess` type: https://docs.python.org/3/library/subprocess.html#subprocess.CompletedProcess

Which is not a valid argument type to `sys.exit` down below.